tests/meta/transfer/test_external_data.py

```diff
@@ -116,6 +117,17 @@ def test_get_all_fastq(external_data_api: ExternalDataAPI, external_data_directo
         assert all([tmp.suffixes == [".fastq", ".gz"] for tmp in files])
 
 
+def test_get_failed_samples(external_data_api: ExternalDataAPI, fastq_file: Path):
+    bad_md5sum_file_path: Path = fastq_file.parent.joinpath("fastq_run_R1_001.fastq.gz")
+    # GIVEN a list of paths with one fastq_file with a correct md5sum and one with an incorrect md5sum
+    # When the failed samples are extracted
+    failed_samples: List[Path] = external_data_api.get_failed_samples(
```

Comment:
```suggestion
    failed_paths: List[Path] = external_data_api.get_failed_samples(
```
You see! The function name trick you into naming the return variable incorrectly. You are getting paths for a sample not samples.